wp-content/themes/pub/wporg-learn-2020/css/components/_page-lesson-plan.scss

```diff
@@ -0,0 +1,137 @@
+.lesson-plan {
+	&-taxonomy {
+		&-header {
+			width: 100%;
+			flex-basis: 100%;
+			font-weight: 400;
+			color: #555D66;
+			font-size: 1.375rem;
+			line-height: 1.36;
+			margin-bottom: 0.75rem;
+		}
+
+		&-description {
+			margin-bottom: 2rem;
+			width: 100%;
+			flex-basis: 100%;
+			color: #555D66;
+		}
+
+		.card-grid  {
+			margin: 2rem -0.5rem;
+			justify-content: flex-start;
+
+			.card {
+				text-align: center;
+				margin-top: 0;
+				margin-bottom: 1rem;
+
+				a:hover,
+				a:focus {
+					text-decoration: none;
+				}
+
+				.dashicons {
+					width: 80px;
+					height: 80px;
+					font-size: 80px;
+					margin: 0 auto 32px;
+					color: #606A73;
+				}
+
+				.taxonomy-title {
+					margin-bottom: 0;
+					font-weight: 700;
+
+					a {
+						color: #555D66;
+						text-decoration: none;
+					}
+				}
+			}
+		}
+
+		&.categories {
+			.card-grid  {
+				margin-top: 3rem;
+			}
+		}
+
+		&.last-grid {
```

Comment:
I think if the top and bottom margin were applied to the taxonomy `row` rather than `.card-grid` we could apply a consistent margin which would collapse between rows, and we wouldn't need anything custom for this last one:

```
&-taxonomy {
    margin: 3rem 0;

    .card-grid  {
        margin: 0 -0.5rem;
    }
}
```